src/settings/OrderNumberForm.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+
+import stripesForm from '@folio/stripes/form';
+
+import {
+  Accordion,
+  AccordionSet,
+  Button,
+  Checkbox,
+  Col,
+  Headline,
+  Pane,
+  Row,
+} from '@folio/stripes/components';
+
+class OrderNumberForm extends Component {
+  static propTypes = {
+    stripes: PropTypes.shape({
+      store: PropTypes.object.isRequired
+    }),
+    handleSubmit: PropTypes.func.isRequired,
```

Comment:
actually since you're not overriding functionality, you can pass you controls as a children, without wrapping them into a `<form>`. What do you think?